simple_classifier/Classifiers/Profiler.py

```diff
@@ -0,0 +1,10 @@
+from simple_classifier.Classifiers.ClassifierProfiler import ClassifierProfiler
+
+
+class Profiler(ClassifierProfiler):
+    def __init__(self, metrics):
+        self.metrics = metrics
+
+    def run(self, classifier):
+        # Run profiler on given classifier
+        pass
```

Comment:
```suggestion
class Profiler:
    def __init__(self, metrics: [str]):
        self.metrics = metrics

    def run(self):
        # Run profiler on given classifier
        pass
```

Alright maybe there's a misunderstanding of the class UML diagram. Please notice that `Splitter` is connected with `SplitterDataset` by a diamond line instead of an arrow line. Hollow diamond indicates that `SplitterDataset` aggregates `Splitter`.

I'll pause my review here. Please revise all aggregations.

Reply:
Could you please explain the removal of the parameter in run()?

Reply:
fixed

Reply:
Since `ClassifierProfiler` holds an attribute of a list of `Profiler` and calls the `run()` method, the data that `Profiler` needs would be provided by its owning `ClassifierProfiler` instance, so I think it duplicates a little to add a classifier parameter.

We could also remove the `Profiler` attribute from `ClassifierProfiler` and keep this parameter to support a plug-and-play style.